lib/omnibus/metadata.rb

```diff
@@ -212,7 +212,7 @@ def truncate_platform_version(platform_version, platform)
     #
     def initialize(package, data = {})
       @package = package
-      @data    = data.dup.freeze
```

Comment:
Hrm. I'm hesitant to unfreeze this. Can we keep it frozen and then `.dup` the `to_hash`? That will unfreeze it